lib/chef_zero/endpoints/organization_user_endpoint.rb

```diff
@@ -20,29 +20,7 @@ def delete(request)
         json_response(200, ChefData::DataNormalizer.normalize_user(user, request.rest_path[3], ['username'], server.options[:osc_compat]))
       end
 
-      def post(request)
```

Comment:
Does the Chef Server support both endpoints?  `POST /organizations/ORGNAME/users` and `POST /organizations/ORGNAME/users/USER`?  If we want chef-zero to emulated CS12 I would think we want want it to do whatever CS12 does.  I'm not sure when the endpoint was changed so I don't know when the old one was deprecated.